code/modules/mob/living/carbon/human/species_types/vampire.dm

```diff
@@ -20,35 +20,40 @@
 	skinned_type = /obj/item/stack/sheet/animalhide/human
 	var/info_text = "You are a <span class='danger'>Vampire</span>. You will slowly but constantly lose blood if outside of a coffin. If inside a coffin, you will slowly heal. You may gain more blood by grabbing a live victim and using your drain ability."
 	var/obj/effect/proc_holder/spell/targeted/shapeshift/bat/batform //attached to the datum itself to avoid cloning memes, and other duplicates
+	var/halloween_version = FALSE //SKYRAT EDIT: Allow a neutered version of vampires without batform
 
 /datum/species/vampire/check_roundstart_eligible()
-	if(SSevents.holidays && SSevents.holidays[HALLOWEEN])
-		return TRUE
-	return FALSE
+	//if(SSevents.holidays && SSevents.holidays[HALLOWEEN]) SKYRAT EDIT - ALL YEAR :smiling_imp:
+	return TRUE
+	//return FALSE SKYRAT EDIT - ALL YEAR :smiling_imp:
```

Comment:
Something else to do in config, it would just need to be commented out entirely instead.